package.json

```diff
@@ -28,11 +28,7 @@
     "test:ci": "run-s test:ci:*",
     "test:dev:ava": "ava",
     "test:ci:ava": "nyc -r lcovonly -r text -r json ava",
-    "prepublishOnly": "run-s prepublishOnly:*",
-    "prepublishOnly:checkout": "git checkout main",
-    "prepublishOnly:pull": "git pull",
-    "prepublishOnly:install": "npm ci",
-    "prepublishOnly:test": "npm test"
+    "prepublishOnly": "npm ci && npm test"
```

Comment:
`run-s` won't work in the GitHub action since it doesn't have dependencies installed yet